.github/workflows/promote_branch.yaml

```diff
@@ -0,0 +1,42 @@
+---
+name: Promote branch
+on:
+  workflow_dispatch:
+    inputs:
+      target-branch:
```

Comment:
Can we use `type: choice` here and for the other ones as well? I think that will make it easier and less error-prone.

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-of-onworkflow_dispatchinputs

Reply:
Tested here https://github.com/johnbieren/release-service-catalog/actions/runs/7584649079/job/20658787534
Added